app/views/comments/edit.js.erb

```diff
@@ -0,0 +1 @@
+$("#<%= dom_id(@comment) %>").replaceWith("<%= j(render "comments/form", comment: @comment) %>");
```

Comment:
good job! This is a nitpik, but next time creating variables for your jquery will make everything easy to follow.